samples/drivers/led_max7219/src/main.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Jimmy Ou <yanagiis@gmail.com>
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/led.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/zephyr.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(app);
```

Comment:
please define app level log level. don't hardcode it.

Reply:
OK. I will change it. Thanks.